a.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
```

Comment:
I am not sure this file is supposed to be included?

Reply:
My mistake